### PR TITLE
Use testing.T.Log for logging from tests

### LINKED ISSUE #26: Use testing.T.Log for logging from tests
This will help reduce too much information logged by tests execution.

### PR DESCRIPTION
**Pull request by hristobojilov**
[#26] Use testing.T.Log for logging from tests 

Signed-off-by: Hristo Bozhilov <Hristo.Bozhilov@bosch.io>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -28,6 +28,7 @@ require (
 	github.com/lithammer/shortuuid/v3 v3.0.4 // indirect
 	github.com/oklog/ulid v1.3.1 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/stretchr/objx v0.1.1 // indirect
```
Comment (e-grigorov) [inline go.mod:31]: Why do we have this new dependency? Is it test dependency or runtime one?

File: testutil/logger.go [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+const (
+	logFlags int = log.LstdFlags | log.Lmicroseconds | log.Lshortfile
+)
+
+// LogFacade interface allows access to runtime logger
+type LogFacade interface {
+	Log(args ...interface{})
+}
+
+type testexporter struct {
```
Comment (e-grigorov) [inline testutil/logger.go]: It should be: testExporter.

File: testutil/logger_test.go [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+	m.Called(args...)
+}
+
+func TestNewLogger(t *testing.T) {
+	f := new(logFacadeMock)
+	f.On("Log", "[testutil]", "INFO  ", "Simple message")
+
+	l := testutil.NewLogger("testutil", logger.TRACE, f)
+	l.Info("Simple message\n", nil)
+	assert.NotNil(t, l)
```
Comment (e-grigorov) [inline testutil/logger_test.go:37]: This assert should be before l.Info(...)

**Review by e-grigorov — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (hristobojilov) [reply go.mod:31]: This is testcase dependency only.

**Review by e-grigorov — APPROVED** [review]

**Pull request merged**